__tests__/pages/profile/username.test.js

```diff
@@ -8,14 +8,10 @@ import { MockedProvider } from '@apollo/client/testing'
 import dummyLessonData from '../../../__dummy__/lessonData'
 import dummySessionData from '../../../__dummy__/sessionData'
 import { useRouter } from 'next/router'
-import expectLoading from '../../utils/expectLoading'
 
 describe('user profile test', () => {
   const { query } = useRouter()
   query['username'] = 'fake user'
-  test('Should render loading spinner if data is not ready', () => {
-    expectLoading(<UserProfile />)
```

Comment:
Why this test was removed? It shouldn't cause any problems.

Reply:
I'm putting it back. Thought we were getting rid of those, but I must have misunderstood.

Reply:
We removed snapshot testing (rendering full page and comparing element by element). `expectLoading` simply checks for `loading` message in the document, so it should be fine.